misk-policy/src/main/kotlin/misk/policy/opa/RealOpaPolicyEngine.kt

```diff
@@ -89,11 +89,18 @@ class RealOpaPolicyEngine @Inject constructor(
       throw IllegalArgumentException("Must specify document")
     }
 
-    val response = opaApi.queryDocument(document, inputString, provenance).execute()
-    if (!response.isSuccessful) {
-      throw PolicyEngineException("[${response.code()}]: ${response.errorBody()?.string()}")
+    try {
+      val response = opaApi.queryDocument(document, inputString, provenance).execute()
```

Comment:
queryDocument returns a retrofit2.Call<ResponseBody>. The docs say:

> Calls may be executed synchronously with execute, or asynchronously with enqueue. In either case the call can be canceled at any time with cancel. A call that is busy writing its request or reading its response may receive a IOException; this is working as designed.

Since execute() is synchronous, so it will be done by the time this exectues.